From b28d84d0e448a9bdfb26c21ec9c825f3c9030ef2 Mon Sep 17 00:00:00 2001
From: Noriko Hosoi <nhosoi@redhat.com>
Date: Tue, 5 May 2015 14:11:45 -0700
Subject: [PATCH 320/323] Ticket #48180 - Lowering the log level of "Configured
 SSL version range" message (1.2.11 only)

Description: The Configured SSL version range message is logged only
if the log level LDAP_DEBUG_CONFIG (0x00040	== 64) is set.

https://fedorahosted.org/389/ticket/48180

Reviewed by rmeggins@redhat.com (Thank you, Rich!!)

(cherry picked from commit bd0c50ae464291b4d1484c621957a701acd6ac71)
---
 ldap/servers/slapd/ssl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/slapd/ssl.c b/ldap/servers/slapd/ssl.c
index 5451d6e..529dbc6 100644
--- a/ldap/servers/slapd/ssl.c
+++ b/ldap/servers/slapd/ssl.c
@@ -1375,7 +1375,7 @@ int slapd_ssl_init2(PRFileDesc **fd, int startTLS)
         slapdNSSVersions.max = NSSVersionMax;
         (void) slapi_getSSLVersion_str(slapdNSSVersions.min, mymin, sizeof(mymin));
         (void) slapi_getSSLVersion_str(slapdNSSVersions.max, mymax, sizeof(mymax));
-        slapi_log_error(SLAPI_LOG_FATAL, "SSL Initialization",
+        slapi_log_error(SLAPI_LOG_CONFIG, "SSL Initialization",
                         "Configured SSL version range: min: %s, max: %s\n",
                         mymin, mymax);
         sslStatus = SSL_VersionRangeSet(pr_sock, &slapdNSSVersions);
-- 
1.9.3

